tests/test/export/test.sh

```diff
@@ -19,7 +19,7 @@ rlJournalStart
         cmd="tmt tests export $tname"
         rlPhaseStartTest "$cmd"
             rlRun -s "$cmd" 0 "Export test"
-            rlAssertGrep "- name: $tname" $rlRun_LOG
```

Comment:
Note to myself: why did the order change? export still begins with `dict(name=self.name)`, `name` should have been the first item logged. With Python 3.7, keys should be presented in the inserted order, and I did not change things that much...

Reply:
We still support `el8` with `python 3.6` so... we probably should not rely on the order.

Reply:
Things are fine. The test checks fmf id, and I did indeed change the order of fields. In any case, it's an export, and YAML on top of that, `grep` and presuming any particular order of fields will be always unstable. Dropping `-` should be perfectly safe.